contrib/test/travis/check_all_examples_use_standard_init.sh

```diff
@@ -1,16 +1,16 @@
 #!/bin/bash
-FBASE_SNIPPET=test/travis/firebase_init.cc.snip
-FBASE_NOAUTH_SNIPPET=test/travis/no_firebase_init.cc.snip
-FBASE_NOCONFIG_SNIPPET=test/travis/no_config_init.cc.snip
-for example in `find examples/ -name *.ino`;
+FBASE_SNIPPET=contrib/test/travis/firebase_init.cc.snip
```

Comment:
I meant this to be a file-comment, it is the entire script that seems part of the core project.